systemtest/src/test/java/io/strimzi/systemtest/upgrade/StrimziUpgradeST.java

```diff
@@ -252,6 +236,16 @@ private void performUpgrade(JsonObject testParameters, int produceMessagesCount,
             LOGGER.info("Going to deploy KafkaTopic from: {}", kafkaTopicYaml.getPath());
             cmdKubeClient().create(kafkaTopicYaml);
         }
+        // Create bunch of topics for upgrade if it's specified in configuration
+        if (testParameters.getBoolean("bunchTopicsUpgrade")) {
```

Comment:
I don't mean to sound pedantic but `bunchTopicsUpgrade` as a parameter is a little meaningless without context, maybe `generateTopics` would be a little more informative and means we can re-use it in other tests if need be.

I wonder also if there is any value in making the topic names generated a little more random, if this is a check that data is persisted, it might be worth making more problematic topic names to check those also behave  - but i admit that might not have too much value.

Reply:
TBH when I wrote it, I was hoping someone will have better name on mind :) I can change it for sure. 

You mean something like you'll pass a specific value as a seed to a method and it will generate a specific list of names which we could use later to validate a test results with the same method call? ... Or we can save each randomly generated topic name (we have method for it) in a list and then validate collected topics against this list. I am just thinking, but the way with randomly generate topics and keep store them for a future validation sounds like a good idea to me.

Reply:
Phew glad it's not unwelcome name change! I think an input for it having a predictable and repeatable set of topics would be useful but perhaps default it to seed using the current unixepoch? That way each test run has a different randomly generated set , this way if weird things happen with a specific topic name we can repeat it and debug why. Again it seems super unlikely that this will ever happen- but making a random topic name that obeys Kafka topic name regex is probably not too hard.

Reply:
Yes, sounds reasonable. Opened https://github.com/strimzi/strimzi-kafka-operator/issues/3194

Reply:
Cool, were you planning on changing the param name prior to merge?

Reply:
bunchTopicsUpgrade -> generateTopics

Reply:
yes, going to do it right now

Reply:
Done :) thanks for reminding me